src/foam/nanos/auth/PermissionedPropertyDAO.js

```diff
@@ -48,6 +48,9 @@ foam.CLASS({
       name: 'select_',
       javaCode: `
       if ( x.get("auth") != null ) {
+        if ( predicate != null && ! predicate.authorize(x) ) {
+          throw new AuthenticationException("You do not have access to the querying property / properties");
```

Comment:
need to adjust the wording. 
"Property access restricted."  or "Property access denied", or something along those lines. 
@nanoscott thoughts on this? This is when a query is made against properties that the user does not have 'read' access to.

Reply:
"Access Denied: Protected property" should work. You want to be clear on the cause.

Reply:
"Access Denied: Property protected" Or "Access Denied: Query on property not allowed"

Reply:
Updated exception message.